Services/UI/OJS.Services.Ui.Business/Cache/Implementations/ContestParticipantsCacheService.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace OJS.Services.Ui.Business.Cache.Implementations;
+
+using OJS.Services.Infrastructure.Constants;
+using OJS.Services.Infrastructure.Cache;
+using OJS.Services.Common.Models.Cache;
+using System.Threading.Tasks;
+
+public class ContestParticipantsCacheService : IContestParticipantsCacheService
+{
+    private readonly ICacheService cache;
+    private readonly IParticipantsBusinessService participantsBusinessService;
+
+    public ContestParticipantsCacheService(ICacheService cache, IParticipantsBusinessService participantsBusinessService)
+    {
+        this.cache = cache;
+        this.participantsBusinessService = participantsBusinessService;
+    }
+
+    public Task<ContestParticipantsViewModel> GetCompeteContestParticipantsCount(int contestId, int? cacheSeconds = CacheConstants.FiveMinutesInSeconds)
+        => cacheSeconds.HasValue
```

Comment:
`cacheSeconds` should always have value (either provided from outside or default). Make it non-nullable in the parameter:
```
, int cacheSeconds = CacheConstants.FiveMinutesInSeconds)
```